ui/apps/platform/src/utils/dateUtils.test.ts

```diff
@@ -17,3 +26,87 @@ describe('dateUtils', () => {
         });
     });
 });
+
+describe('displayDateTimeAsISO8601', () => {
+    it('should return compliant ISO string with Z', () => {
+        const date = new Date('2024-01-01T12:34:56Z');
```

Comment:
Following the thought from the channel, what are pro and con of difference month and day?

Reply:
Great idea, I'll change these so they are not the same.